### ISSUE TITLE
[Bug]: Tharaphi <=> သရဖီ

### ISSUE THREAD

**Issue by tintnaingwinn** [ageekdev/myanmar-name-converter#20]
### What happened?

**Convert to Myan-glish Name**
input: မေသရဖီ
response result: May Tha Ya Phy
expected result: May Tharaphi

**Convert to Myanmar Name**
input: May Tharaphi
response result:  မေ
expected result: မေသရဖီ

### How to reproduce the bug

**Convert to Myan-glish Name**
```php
$name = 'မေသရဖီ';
MMName::convertToEn($name);
```

**Convert to Myanmar Name**
```php
$name = 'May Tharaphi'
MMName::convertToMm($name);
```

### Package Version

1.x

### PHP Version

8.2

### Laravel Version

10.x

### Which operating systems does with happen with?

macOS

**Comment by thantthuhein-genie** [comment]
Fixed in version 1.1.0

**Comment by thantthuhein-genie** [comment]
Fixed in version 1.1.0
